package.json

```diff
@@ -13,12 +13,12 @@
     "build": "webpack --config webpack.config.js"
   },
   "peerDependencies": {
-    "bl": "^4.0.2",
     "cross-spawn": "^7.0.3"
   },
   "devDependencies": {
     "@types/bl": "^2.1.0",
     "@types/cross-spawn": "^6.0.2",
+    "bl": "^4.0.2",
```

Comment:
Maybe I am missing something but looking back at this we use `BufferList` from `bl` in the `[processManager](https://github.com/vleandersson/simple-nodejs-threader/blob/a474d212ed4f705314d36f3bb777c9a2d280db73/src/processManager.ts#L2)` . This means that  in order to run this package in any production code, you will require to build with peer-deps in order for the `promiseSpawn` to work.

We could make sure we remove the usage (as it is fairly small), but it is unclear for me the value in this as peer-deps should not be installed by default - you have to manually add these.

Reply:
Thank you for explaining @vleandersson, that makes sense